recipe/meta.yaml

```
@@ -32,7 +34,7 @@ requirements:
     - wheel
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy <1.26
```

```suggestion
    - {{ pin_compatible('numpy', upper_bound='1.26') }}
```
This is a better variant

---

awesome, I was looking for that!